corehq/apps/cleanup/management/commands/check_case_integrity.py

```diff
@@ -1,106 +0,0 @@
-from collections import defaultdict
```

Comment:
I don't think this script is useful anymore (and it's a bit of a mess). easier to just remove it then try and figure out how to port it over.